doc/conf.py

```diff
@@ -30,7 +31,7 @@
     'PROJECT': project,
     'DESCRIPTION': description,
     'AUTHOR': authors,
-    # will work without this - for canonical (so can ignore when building locally or test deploying)    
```

Comment:
Any idea what "for canonical" means?

Reply:
nope :(

Reply:
The canonical url, for search engines.

Same text appears in multiple projects. Maybe a bit late now, but the original source of the note is https://github.com/pyviz/nbsite/blob/master/nbsite/tmplate2/conf.py#L39 (template used to start a new project).

Reply:
Ok, thanks.  @jsignell, please change the comment to "WEBSITE_SERVER is optional for tests and local builds, but allows defining a canonical URL for search engines"